HW3/Task1.py

```diff
@@ -0,0 +1,21 @@
+'''
+you have a list of elements [1, 2, 3, 4, 5, 6, 7, 8]. Loop through the list using the foreach loop. The element with an
+odd index is put into a new list of tuples where the first element is the index and the second is the value. [(index,
+value)]. accordingly, elements with an even index are placed in another list of tuples with the same format as in the
+case with odd indices.
+'''
+
+list_of_elements = [1, 2, 3, 4, 5, 6, 7, 8]
+odd_list_of_tuples = []
+even_list_of_tuples = []
+index = 0
+
+for number in list_of_elements:
+    if index % 2 != 0:
+        odd_list_of_tuples.append((index, number))
+    else:
+        even_list_of_tuples.append((index, number))
+    index += 1
```

Comment:
it works but it is not an optimal solution. In the lesson I show smth that can help solve this problem in more pythonic way